modules/cluster-logging-release-notes-5.4.0.adoc

```diff
@@ -40,7 +40,7 @@ link:https://access.redhat.com/errata/RHSA-2022:1461[{logging-title-uc} Release
 
 * Before this update, clusters with a large number of namespaces caused Elasticsearch to stop serving requests because the list of namespaces reached the maximum header size limit. With this update, headers only include a list of namespace names, resolving the issue.	(link:https://issues.redhat.com/browse/LOG-1899[LOG-1899])
 
-* Before this update, the OpenShift Logging dashboard showed the number of shards 'x' times bigger than actual value when Elasticsearch has 'x' nodes. This was because it was printing all primary shards for each ES pod and processing sum on it, while the output is always for the whole ES cluster. With this update, the calculation has been corrected.	(link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])	
+* Before this update, the OpenShift Logging dashboard showed the number of shards 'x' times bigger than actual value when Elasticsearch has 'x' nodes. This was because it was printing all primary shards for each ES pod and processing sum on it, while the output is always for the whole ES cluster. With this update, the calculation has been corrected.	(link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])
```

Comment:
```suggestion
* Before this update, the OpenShift Logging dashboard showed the number of shards 'x' times bigger than the actual value when Elasticsearch has 'x' nodes. This error happened because it was printing all primary shards for each ES pod and processing sum on it, while the output is always for the whole ES cluster. With this update, the calculation has been corrected. (link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])
```